extensions-contrib/dropwizard-emitter/src/main/java/org/apache/druid/emitter/dropwizard/DropwizardEmitterConfig.java

```diff
@@ -0,0 +1,136 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.druid.emitter.dropwizard;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Objects;
+
+
+public class DropwizardEmitterConfig
+{
+  private static int DEFAULT_MAX_GAUGE_COUNT = 100_000;
+  @JsonProperty
+  private final List<DropwizardReporter> reporters;
+  @JsonProperty
+  private final String prefix;
+  @JsonProperty
+  private final Boolean includeHost;
+  @JsonProperty
+  private final String dimensionMapPath;
+  @JsonProperty
+  private final List<String> alertEmitters;
+  @JsonProperty
+  private final int maxGaugeCount;
+
+  @JsonCreator
+  public DropwizardEmitterConfig(
+      @JsonProperty("reporters") List<DropwizardReporter> reporters,
+      @JsonProperty("prefix") String prefix,
+      @JsonProperty("includeHost") Boolean includeHost,
+      @JsonProperty("dimensionMapPath") String dimensionMapPath,
+      @JsonProperty("alertEmitters") List<String> alertEmitters,
+      @JsonProperty("maxGaugeCount") Integer maxGaugeCount
+  )
+  {
+    this.reporters = reporters;
```

Comment:
null/empty check ? it would be useless to set it up with no reporters and probably a configuration error.